analyses/chromosomal-instability/util/chr-break-plot.R

```diff
@@ -115,7 +115,71 @@ multipanel_break_plot <- function(granges_list,
     base_height = 7,
     base_width = 20
   )
+}
 
-  # Print out the plot while we are here
-  full_plot
+breaks_cdf_plot <- function(density_file, metadata_df, cdf_plot_file) {
+  # Given a genome wide breaks density file path, plot the CDF distribution 
+  # for it by histology.
+  #
+  # Args:
+  #   density_file: a chromosomal breaks density file path where each sample is
+  #                 a row with these columns: samples, experimental_strategy,
```

Comment:
Is experimental_strategy required here? Seems like only samples and breaks_count is used in this function.

Reply:
Oh you are right. My comment is wrong.